content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -291,6 +308,32 @@ Here, we first check that the optional parameter is present with `!has(params.op
   evaluated, and optionalNumber will be checked to ensure that it contains a value between 5 and
   10 inclusive.
 
+
+##### Per-Namespaced Parameters
```

Comment:
```suggestion
##### Per-namespaced parameters
```

Reply:
It should be "Namespaced parameters" or "Per-namespace parameters".